pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/WhenStageTest.java

```diff
@@ -191,19 +191,72 @@ public void whenChangesetMoreCommits() throws Exception {
 
         builder.logContains("Hello", "JS World")
                 .logNotContains(
-                        "Stage 'Two' skipped due to when conditional",
+                        "Stage \"Two\" skipped due to when conditional",
                         "Warning, empty changelog.",
                         "Examining changelog from all builds of this change request.")
                 .go();
     }
 
+    @Test
+    public void whenChangesetPR() throws Exception {
```

Comment:
Related?

Reply:
Merge/rebase fun. Ignore for now, won't be in the diff for the real PR once #227 has merged.